config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'home#index'
+  root 'home#index' 
+  get "/pages/:page" => "pages#show"
```

Comment:
Board is a function of the game, so we would want this on games#show & /games/:id